Test/Framework/Vector2Test.cs

```diff
@@ -26,53 +31,53 @@ public void Multiply()
             var vector = new Vector2(1, 2);
 
             // Test 0.0 scale.
-            Assert.AreEqual(Vector2.Zero, 0 * vector);
-            Assert.AreEqual(Vector2.Zero, vector * 0);
-            Assert.AreEqual(Vector2.Zero, Vector2.Multiply(vector, 0));
-            Assert.AreEqual(Vector2.Multiply(vector, 0), vector * 0.0f);
```

Comment:
Yes... there was nothing wrong with all the existing tests.  They all passed.

This is also the sort of thing that should have all gone into another PR.  Lumping it in here with a ton of doc changes just complicated things further.